docs/Project-Configuration/Project-Commands.md

```diff
@@ -1309,15 +1309,26 @@ and contains these fields:
     bucket: mciuploads
     region: us-east-1
     local_file: src/mongo-binaries.tgz
+# Static credentials (deprecated):
+- command: s3.get
+  params:
+    aws_key: ${aws_key}
+    aws_secret: ${aws_secret}
+    remote_file: ${mongo_binaries}
+    bucket: mciuploads
+    region: us-east-1
+    local_file: src/mongo-binaries.tgz
 ```
 
 Parameters:
 
 -   `aws_key`: your AWS key (use expansions to keep this a secret).
 -   `aws_secret`: your AWS secret (use expansions to keep this a secret).
 -   `aws_session_token`: your temporary AWS session token (use expansions to keep this a secret).
-    Note: If you are generating temporary credentials using `ec2.assume_role`, it is recommended
-    to pass in the role_arn directly to your s3 commands instead.
+    Note: If you are generating temporary credentials using `ec2.assume_role`, you can instead
+    pass in the role_arn directly to your s3 commands. If you have multiple S3 commands in quick
+    succession, it is recommended you call ec2.assume_role once and pass in the credentials
```

Comment:
nit: \`ec2.assume_role\` to be consistent with the earlier sentence in the note.

Same thing in the doc below.